mux_test.go

```diff
@@ -336,8 +336,8 @@ func TestPath(t *testing.T) {
 			shouldMatch:  false,
 		},
 		{
-			title:        "Path route, URL with encoded slash does match",
-			route:        new(Route).Path("/v1/{v1}/v2"),
+			title:        "Path route with useEncodedPath, URL with encoded slash does match",
+			route:        (&Route{useEncodedPath: true}).Path("/v1/{v1}/v2"),
```

Comment:
I don't really like this. I'm debating whether it's worth breaking out into something like TestSkipClean